v3/src/data-interactive/iframe-phone-types.ts

```diff
@@ -0,0 +1,3 @@
+// TODO any type. This is a message sent via iframe-phone. iframe-phone says it should be a string, but that's
+// just not true. Hopefully iframe-phone will provide an accurate type in the future.
```

Comment:
We should add a PT story to the CODAP v3 PT board about updating types in iframe-phone. We're the ones running into it right now so that makes us the logical team to fix it.